DAY4/zxxng/app.js

```
@@ -0,0 +1,6 @@
+const toggleBtn = document.querySelector('.nav-toggle');
+const hide = document.querySelector('.hide');
+
+toggleBtn.addEventListener('click', () => {
+  hide.classList.toggle('hide');
+});
```

default 상태가 서브메뉴들이 숨겨져 있어 변수명을 hide로 지정하신 것 같은데, 

`classList.toggle('className')` 자체가 'className'이 존재한다면 클래스를 제거하고, 존재하지 않다면 클래스를 추가하는 메서드다 보니까

이 모두를 아우를 수 있는 변수명이 코드를 이해하기 더 편할 것 같습니다.
예를 들면 `subMenu`??

---

해당 클래스의 역할이 ul 태그를 숨기거나 보이게 하는 기능을 하고 있어 클래스명을 `hide`로 지정했습니다. 
```js
// html
<ul class="menu hide">

// css
.hide {
  display: none;
}
```
이미 menu라는 클래스명은 기본으로 부여되어 있어 sub-menu를 클래스명으로 한다면 오히려 더 헷갈릴것 같다는 생각이 듦니다..!